docs/kb/api-version-3/export.md

```diff
@@ -39,11 +39,11 @@ A procedure is a query and execution plan used to retrieve the data. Each object
 
```

Comment:
General note about the deleted args: 

"Selects list membership based on if it is deleted or not. The value can be `true`, `false`, or `all`"

If you're referring to an object's records here, I'd consider rephrasing all of these to something like "Selects list membership records based on whether they are deleted. The value can be ...."